### ISSUE TITLE
Tracker link text limited to 30 chars

### ISSUE THREAD

**Issue by jobst** [salesagility/SuiteCRM#6171]
#### Issue

If you create a tracker when editing templates the tracker link text is limited to 30 chars. If you need  longer than that the tracker is saved but will never work as the entry is chopped off when saving to the database.
This took a while to figure out as the URL is correct, it shows correctly in the tracker list, it only shows when looking at Overview of a campaign, there you see the tracker_name is chopped, which alerted me to look at the DB and the entry length of varchar.

#### Expected Behavior

The tracker link text should be allowed to be longer as it is not possible to link to quotes or longer text. These days it is normal wanting to link to longer text.

#### Actual Behavior

the tracker link text is chopped, therefor the link stops working.

#### Possible Fix

Change the size for tracker_name from varchar(30) to varchar(100).

#### Steps to Reproduce

1. edit a campaign template with some long text in it and highlight something longer than 30 chars
2. create a tracker
3. enter the text that is longer than 30 chars
4. save
5. the tracker name is chopped ohf but the link href refers to the longer text
6. link text will fail coz it cannot find the tracker name in db

#### Context

makes it impossible to assign a tracker link text longer than 30

#### Your Environment

* SuiteCRM Version used: 7.9.6
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): N/A
* Environment name and version (e.g. MySQL, PHP 7): N/A
* Operating System and version (e.g Ubuntu 16.04): N/A

**Comment by daniloff200** [comment]
I'm not sure, but, looks like that should be closed, because tracker_name length is now 255.
According to https://github.com/salesagility/SuiteCRM/blob/153b2bae76097cdba9fc9c025bcd829a702b8687/modules/CampaignTrackers/vardefs.php

**Comment by jobst** [comment]
> I'm not sure, but, looks like that should be closed, because tracker_name length is now 255.
> According to https://github.com/salesagility/SuiteCRM/blob/153b2bae76097cdba9fc9c025bcd829a702b8687/modules/CampaignTrackers/vardefs.php

I don't think that was ever the problem, it was storing it in the DB when parts of the URL were chopped off - the fact that I could ENTER a longer URL into the input fields supports this.
MYSQL will not report an error when the string is too long, it will simply "chop it".
